### ISSUE TITLE
precondition-sql-check raises liquibase.exception.ChangeLogParseException

### ISSUE THREAD

**Issue by Alenichel** [liquibase/liquibase#2918]
## Environment

**Liquibase Version**: 4.11.0

**Liquibase Integration & Version**: CLI

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**: Any

**Operating System Type & Version**: MacOS 12.0.1

## Description
Any SQL changeset with a precondition-sql-check fails

## Steps To Reproduce

- Create a SQL changeset with a precondition-sql-check and run any CLI commands (e.g. liquibase update or liquibase status)

## Actual Behavior
`
Caused by: liquibase.exception.ChangeLogParseException: liquibase.exception.SetupException: 
Unexpected formatting at line 846. Formatted SQL changelogs require known formats, such as '--property name=<property name> value=<property value>' and others to be recognized and run. Learn all the options at https://docs.liquibase.com/concepts/changelogs/sql-format.html
	at liquibase.parser.core.xml.AbstractChangeLogParser.parse(AbstractChangeLogParser.java:25)
	at liquibase.Liquibase.getDatabaseChangeLog(Liquibase.java:369)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:224)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.Liquibase.runInScope(Liquibase.java:2405)
	at liquibase.Liquibase.update(Liquibase.java:211)
	at liquibase.Liquibase.update(Liquibase.java:197)
	at liquibase.integration.commandline.Main.doMigration(Main.java:1869)
	at liquibase.integration.commandline.Main$1.lambda$run$0(Main.java:398)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.integration.commandline.Main$1.run(Main.java:397)
	... 24 more
`

The line referred in the error message is the third line of the following changeset causing the issue:

```
-- changeset author:name context:XXX
-- preConditions onFail:MARK_RAN onError:MARK_RAN
-- precondition-sql-check expectedResult:0 SELECT COUNT(*) from TABLE where CODE = 'CODE'
INSERT INTO TABLE (ID, CODE, DESCRIPTION) VALUES (SEQ_NAME.nextVal, 'CODE', 'DESCRIPTION');

```

This changeset, and all other affected, had always run until now (before updating to 4.11.0 fro 4.9.1) and it seems to me that they strictly follow the example available at: 
https://docs.liquibase.com/concepts/changelogs/sql-format.html

## Expected/Desired Behavior
The error is no more raised

**Comment by vcotugno** [comment]
Same here on Windows 10/11.
Latest version working with precondition-sql-check is 4.9.1

**Comment by NorisM96** [comment]
Same here on Linux on latest liquibase version, had to revert to previous version

**Comment by nvoxland** [comment]
I think this was fixed by #3037 in 4.14.0. I can't replicate the problem with the current build.

**Comment by vcotugno** [comment]
i can confirm that the issue is fixed as of release 4.14.0, thanks
